tests/sls/common/timezone.py

```diff
@@ -0,0 +1,18 @@
+import subprocess
+
+from tests.util import Failure, Success
+
+
+def run():
+    ret = subprocess.run(
+        ['date'],
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE
+    )
+
+    stdout = ret.stdout.decode('utf-8')
+
+    if ret.returncode == 0 and 'UTC' in stdout:
+        return Success('Date is in UTC')
```

Comment:
Should I put the brew python install inside a saltscript or inside the .travis/install...sh ?